readme.md

```diff
@@ -78,6 +78,21 @@ Flags:
       --source string     Path to 'acme.json' file. (default "./acme.json")
 ```
 
+### Configuration of Key/Value stores
+
+#### Consul
+
+Consul connection parameters can be set via environment variables, see https://www.consul.io/docs/commands/index.html#environment-variables.
+
+#### Etcd
+
+Version 3 is not supported.
```

Comment:
could you explain why?

Reply:
I wanted to write "Version 2" is not supported because it is marked as deprecated in [traefik](https://docs.traefik.io/configuration/backends/etcd/).